spyder/widgets/shortcuts.py

```diff
@@ -0,0 +1,140 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright © 2009- The Spyder Development Team
+# Licensed under the terms of the MIT License
+# (see spyderlib/__init__.py for details)
+
+"""Shortcut Summary dialog"""
+
+# TODO:
+# calculate windows size from screen size
+
+# Standard library imports
+from operator import itemgetter
+from itertools import groupby
+
+# Third party imports
+from qtpy.QtCore import Qt
+from qtpy.QtGui import QFont
+from qtpy.QtWidgets import (QDialog, QLabel, QGridLayout, QGroupBox,
+                            QVBoxLayout, QHBoxLayout, QDesktopWidget)
+
+# Local imports
+from spyder.config.base import _
+from spyder.config.gui import iter_shortcuts
+
+# Constants
+MAX_FONT_SIZE = 16
+MIN_FONT_SIZE = 8
+
+class ShortCutsSummaryDialog(QDialog):
```

Comment:
Please add a docstring with some information about the dialog, what it contains and what it looks like.